src/plays/quizeo/src/Quizeo.jsx

```diff
@@ -0,0 +1,27 @@
+import { getPlayById } from 'meta/play-meta-util';
+import PlayHeader from 'common/playlists/PlayHeader';
+import './quizeo.css'
+
+function Quizeo(props) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
+        {/* Your Code Starts Here */}
```

Comment:
I dont see anything imported here. What's the entry point to your play?

Reply:
The entry is through routing in the RouteDef JSX file. I added the entry point here but then it needs to add to the routedef JSX file. The app starts with the home page and then the pages are built with the components.